autorest/version.go

```diff
@@ -6,8 +6,8 @@ import (
 
 const (
 	major        = "7"
-	minor        = "0"
-	patch        = "0"
+	minor        = "2"
+	patch        = "3"
```

Comment:
Actually, I messed up with go-autorest and forgot to adjust this for 7.2.3. This should be updated to 7.2.4